app/build.gradle

```
@@ -1,6 +1,7 @@
 plugins {
     id 'com.android.application'
     id 'org.jetbrains.kotlin.android'
+    id 'org.jetbrains.kotlin.plugin.serialization' version('1.7.10')
```

```suggestion
    id 'org.jetbrains.kotlin.plugin.serialization' version('1.7.10')
    id 'kotlin-kapt'
```

아래에 kapt 기능 사용해야되니까 이거 추가해야됨